README.md

```diff
@@ -14,107 +14,105 @@ There are two portions to this curriculum:
-|Implementations|rust-lightning|Incomplete implementation|[Rust-Lightning](https://github.com/rust-bitcoin/rust-lightning) |
-|Implementations|c-lightning||[c-lightning — a Lightning Network implementation in C](https://github.com/ElementsProject/lightning) |
-|Implementations|Eclair||[Eclair - A scala implementation of the Lightning Network](https://github.com/ACINQ/eclair) |
-|Implementations|DotNetLightning|Incomplete implementation|[Lightning network daemon with F#](https://github.com/joemphilips/DotNetLightning) |
-|Implementations|geelightning|Incomplete implementation|[geelightning](https://gitlab.com/nblockchain/geelightning) in Rust |
-|Setting up a node||| [Bitcoin Lightning Network #1: Can I compile and run a node?](https://medium.com/andreas-tries-blockchain/bitcoin-lightning-network-1-can-i-compile-and-run-a-node-cd3138c68c15), [Beginner’s Guide to ️Lightning️ on a Raspberry Pi](https://github.com/Stadicus/guides/blob/master/raspibolt/README.md), [My Lightning Node setup with c-lightning](https://medium.com/coinmonks/my-lightning-node-setup-with-c-lightning-45bbb9993c0), [Fastest and cheapest way to get your own Lightning Node running - on a RaspberryPi with a nice LCD](https://github.com/rootzoll/raspiblitz) |
+| Subjects                 | Topics                                                   | Sub-topics                                          | Sources                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
It's very difficult to see what links have changed here when there's an update to the table structure as well - could you further split out the commits to have the link updates/formatting changes in separate commits?

Reply:
I split in 2 commits, but still GH diff preview does not catch everything. In Rider it is easier to spot the differences:
![image](https://github.com/chaincodelabs/lightning-curriculum/assets/4750755/290adc8f-a207-4e83-abd5-a54fc1ccbf11). Hopefully, it helps.

I think this problem is going to happen again every time this list gets updated. I still think, [this](https://github.com/chaincodelabs/lightning-curriculum/pull/11) restructuring would make it easier to update this list in the future.

Reply:
> A couple of links have been updated

Could you give me a tl;dr on the links that have changed?